### ISSUE TITLE
[Bug] Parameter issue in "az pipelines runs artifact list"

### ISSUE THREAD

**Issue by s-bauer** [Azure/azure-devops-cli-extension#585]
Referring to https://github.com/Azure/azure-cli-extensions/issues/676

I'm trying to get all artifacts for a given buildId. I use the following command:

```
az pipelines runs artifact list --run-id [BuildId] --org "https://dev.azure.com/[MyOrg]" --project "[MyProject]"
```

however it throws the error, that you can see in the referenced issue. However I figured out there's probably a parameter mapping problem. If i switch "--project" and "--run-id" it works:

```
az pipelines runs artifact list --project [BuildId] --org "https://dev.azure.com/[MyOrg]" --run-id "[MyProject]"
```

Please fix this!!

**Comment by gauravsaralMs** [comment]
@s-bauer : can you please share debug logs?
az pipelines runs artifact list --run-id [BuildId] --org "https://dev.azure.com/[MyOrg]" --project "[MyProject]" --debug

**Comment by s-bauer** [comment]
@gauravsaralMs There u go: https://gist.github.com/s-bauer/485dfd250c61656d8b37e6c6db6effe2

**Comment by gauravsaralMs** [comment]
Thanks for reporting this @s-bauer . We will fix this ASAP

**Comment by atbagga** [comment]
Fix is checked in. It should be available in the next release 0.9.0 and later.

**Comment by s-bauer** [comment]
@atbagga When will 0.9.0 be released? Any eta yet?
